tests/vnc_test.go

```diff
@@ -65,7 +65,7 @@ var _ = Describe("[rfe_id:127][crit:medium][arm64][vendor:cnv-qe@redhat.com][lev
 	Describe("[rfe_id:127][crit:medium][vendor:cnv-qe@redhat.com][level:component]A new VirtualMachineInstance", func() {
 		BeforeEach(func() {
 			var err error
-			vmi = libvmi.New(libvmi.WithResourceMemory("1Mi"))
+			vmi = libvmi.New()
```

Comment:
@brianmcarey please use
```suggestion
			vmi = libvmi.NewGuestless()
```